src/Native/build.proj

```diff
@@ -89,6 +89,8 @@
                           RelativePath="Microsoft.ML\runtimes\$(PackageRid)\native" />
       <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)LdaNative$(NativeLibExtension)"
                           RelativePath="Microsoft.ML\runtimes\$(PackageRid)\native" />
+      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MatrixFactorizationNative$(NativeLibExtension)"
```

Comment:
(nit) it would be nice to not have this `Microsoft.ML.MatrixFactorizationNative` asset stuck right in the middle of the `Microsoft.ML` assets. Instead, the assets for each nuget package should be grouped together. That way you can easily read which assets go where.

Reply:
Do you mean create another item group right below the current one? For example,

+    <ItemGroup>
+      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MatrixFactorizationNative$(NativeLibExtension)"
+                          RelativePath="Microsoft.ML.MatrixFactorizationNative\runtimes\$(PackageRid)\native" />
+    </ItemGroup>

Reply:
It doesn't have to be a new `ItemGroup`, but it can be if you want.

My main concern here is that you are injecting an unrelated item in a sequence of related items, which makes it hard to read/discern which native assets are in the `Microsoft.ML` package:

* Microsoft.ML\LdaNative
* Microsoft.ML\FastTreeNative
* Microsoft.ML.MatrixFactorizationNative\MatrixFactorizationNative
* Microsoft.ML\FactorizationMachineNative

Just keep all the `Microsoft.ML` assets grouped together.

Reply:
Ok. I will move it to the last entry of the existing item group. Thanks.